cypress-tests/cypress/e2e/admin/pageBuilder/templates/pageTemplatesExportImport.cy.ts

```diff
@@ -0,0 +1,81 @@
+import { customAlphabet } from "nanoid";
+
+context("Page Builder - Template Export&Import", () => {
+    const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz");
+    const titleString1 = nanoid(6);
+    const titleString2 = nanoid(6);
+    const titleString3 = nanoid(6);
+    const titleString4 = "!#$%&/()=?*";
+    const pageTemplateData1 = {
+        title: titleString1,
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData2 = {
+        title: titleString2,
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData3 = {
+        title: titleString3,
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData4 = {
+        title: titleString4,
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllTemplates();
+        cy.pbCreatePageTemplate(pageTemplateData1);
+        cy.pbCreatePageTemplate(pageTemplateData2);
+        cy.pbCreatePageTemplate(pageTemplateData3);
+        cy.pbCreatePageTemplate(pageTemplateData4);
+    });
+    
+
+    it("Should be able to export templates and then import them again", () => {
+        cy.visit("/page-builder/page-templates");
+        cy.findByTestId("export-template-button").click();
+        cy.findByTestId("pb-templates-export-dialog-export-url")
+            .invoke("text")
+            .then(text => {
+                const url = text.trim();
+                console.log(url);
+                cy.pbDeleteAllBlockCategories();
```

Comment:
Well since the test is importing/exporting templates, after exporting the data we need to clear the UI.
So by running cy.pbDeleteAllBlockCategories() and cy.pbDeleteAllBlocks() we completely remove all data that might interfere with the assertion of the data after importation.